app/views/manual_sections/show.html.erb

```diff
@@ -15,6 +15,15 @@
   </div>
 </div>
 
+<% if @section.body.present? %>
```

Comment:
I think it'd be worth adding an assertion to `viewing_a_manual_section_spec` to check for the body being rendered...